docs/proposals/organizations.md

```diff
@@ -0,0 +1,137 @@
+# Organizations
+
+## Problems
+ 1.  Cluster administrators want to allocate a pool of resources to a subset of their company, and delegate the authority to manage how that pool is assigned to an organization administrator.
+ 2.  An organization administrator needs to be able to limit a particular project's usage of shared quota (think htb rate/ceil for tc qdisc classes).
+ 3.  Cluster administrators want to delegate control over groups of users to an organization administrator - that organization administrator should be able to manage the access of a set of users to individual projects under that organization umbrella.
+ 4.  Cluster administrators want to allow self-service of users on the cluster, but the total allocated resources those self service users can get access to is limited (to prevent abuse / unfair use of resources).
+
+We can do this by introducing the concept of project ownerhsip.  A Project is owned by at most one (Organization xor User).  We'll also introduce the idea of a `ClusterResourceQuota` object that can be associated with an organization or a user. An Organization can have multiple org-owners who are allowed to manage quota allocation to owned projects and OrgGroups (groups that scoped to projects owned by an Organization).
+
+### Open Questions
+ 3.  Can we punt on adding users as organization members?  Still don't see a clear answer on this.
+
+     Organization-admins should not be able to force users into an Org.  Users should not be able to join any Org they want.  I'd like to get the structure correct and punt on the ability to add members.  It would require some sort of invitation/request/approval flow.
+
```

Comment:
What's the downside of allowing org admins to add arbitrary users into an organization? I think CloudFoundry does this for Spaces, which is roughly analogous to our notion of Projects. As long as we provide a way for members to remove themselves, and we ensure that there is always at least one admin, I don't see why a complex invite/request/approval flow is needed.